.idea/libraries/R_User_Library.xml

```diff
@@ -0,0 +1,6 @@
+<component name="libraryTable">
```

Comment:
is all of this .idea stuff needed? couldnt we just add it to .gitignore?

Reply:
yes, this is accurate. Please do not commit IDE project files to the repo